tests/conftest.py

```diff
@@ -256,9 +256,23 @@ def state_with_localhost():
 
 
 @pytest_asyncio.fixture
-async def localhost_receiver(state_with_localhost, event_loop) -> TrapReceiver:
+async def localhost_pysnmp_receiver(state_with_localhost, event_loop) -> pysnmp_backend.TrapReceiver:
```

Comment:
Shouldn't we also use the `unused_udp_port` fixture here?

Reply:
Probably, but the problem wasn't evident with this particular test, which was green the whole time.